### PR TITLE
Changes to example page html for listbox-combo.html and LayoutGrid.html

### PR DESCRIPTION
**Pull request by spectranaut**
Hi @mcking65 -- per our discussion, here is a PR to master with the changes to the example pages themselves. This commit only add "data-test-id" to the appropriate tables.

### CODE REVIEW

File: examples/combobox/aria1.1pattern/listbox-combo.html [hunk 1]
```diff
@@ -274,7 +274,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
               </ul>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-down-arrow">
```
Comment (mcking65) [inline examples/combobox/aria1.1pattern/listbox-combo.html]: This is same ID as line 185, but won't the tests be different?

File: examples/combobox/aria1.1pattern/listbox-combo.html [hunk 2]
```diff
@@ -264,7 +264,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
               </ul>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-escape">
```
Comment (mcking65) [inline examples/combobox/aria1.1pattern/listbox-combo.html]: This is same ID as line 222, but won't tests be different?

File: examples/combobox/aria1.1pattern/listbox-combo.html [hunk 3]
```diff
@@ -284,7 +284,7 @@ <h3 id="kbd_label_listbox">Listbox Popup</h3>
               </ul>
             </td>
           </tr>
-          <tr>
+          <tr data-test-id="key-up-arrow">
```
Comment (mcking65) [inline examples/combobox/aria1.1pattern/listbox-combo.html]: This is same ID as line 199, but won't tests be different?

File: examples/combobox/aria1.1pattern/listbox-combo.html (hunk 1 above)
Comment (spectranaut) [reply examples/combobox/aria1.1pattern/listbox-combo.html]: you are right, and it lead to me to not test this row of this table. Changing this to: `listbox-key-down-arrow` and the other to `textbox-key-down-arrow`

File: examples/combobox/aria1.1pattern/listbox-combo.html (hunk 2 above)
Comment (spectranaut) [reply examples/combobox/aria1.1pattern/listbox-combo.html]: Changing this to: `listbox-key-escape` and the other to `textbox-key-escape`

File: examples/combobox/aria1.1pattern/listbox-combo.html (hunk 3 above)
Comment (spectranaut) [reply examples/combobox/aria1.1pattern/listbox-combo.html]: Changing this to: `listbox-key-up-arrow` and the other to `textbox-key-up-arrow`

**Comment by spectranaut** [comment]
Hey @mcking65 addressed your feedback. I'll have to add a few more tests to listbox-combo.js -- so good catch. I'll add that to the backlog issue #780

**Comment by mcking65** [comment]
looks good

**Pull request merged**